cmd/api/src/auth/model.go

```diff
@@ -149,14 +149,20 @@ func (s authorizer) AllowsAtLeastOnePermission(ctx Context, requiredPermissions
 }
 
 func (s authorizer) AuditLogUnauthorizedAccess(request *http.Request) {
-	// Ignore read logs as they are less likely to occur from malicious access
+	commitId, err := uuid.NewV4()
+	if err != nil {
+		log.Errorf("error generating commit ID for unauthorized access: %s", err.Error())
```

Comment:
This will log the error, but if we don't have a valid uuid, do we really want to try to commit it without one?

Reply:
That's a good point. I'm not sure what the best course of action would be, so leaving it as a pass through to commit without an ID seems sane to me